packages/traceloop-sdk/tests/datasets/test_datasets_operations.py

```
@@ -0,0 +1,120 @@
+@patch.dict("os.environ", {"TRACELOOP_API_KEY": "test-api-key"})
+def test_delete_by_slug_failure():
+    with patch('traceloop.sdk.client.http.HTTPClient') as mock_http_client_class:
+        mock_client = MagicMock()
+        mock_client.delete.return_value = False
+        mock_http_client_class.return_value = mock_client
+
+        datasets = Datasets(mock_client)
+        
+        try:
+            datasets.delete_by_slug("test-dataset")
+            assert False, "Expected exception was not raised"
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)